.github/CODEOWNERS

```
@@ -6,6 +6,7 @@
 *       @lengau
 
 # Documentation owners
+/docs/tutorial/ @medubelko @erinecon
```

```suggestion
/docs/tutorial/ @medubelko @erinecon @jdkandersson 
```
Proposal to also add David as an automatic reviewer.